cmd/nitro/config_test.go

```diff
@@ -15,10 +15,29 @@ import (
 	"time"
 
 	"github.com/offchainlabs/nitro/cmd/genericconf"
+	"github.com/offchainlabs/nitro/cmd/util/confighelpers"
 	"github.com/offchainlabs/nitro/util/colors"
 	"github.com/offchainlabs/nitro/util/testhelpers"
+
+	"github.com/r3labs/diff/v3"
+	flag "github.com/spf13/pflag"
 )
 
+func TestEmptyCliConfig(t *testing.T) {
+	f := flag.NewFlagSet("", flag.ContinueOnError)
+	NodeConfigAddOptions(f)
+	k, err := confighelpers.BeginCommonParse(f, []string{})
+	Require(t, err)
+	var emptyCliNodeConfig NodeConfig
+	err = confighelpers.EndCommonParse(k, &emptyCliNodeConfig)
+	Require(t, err)
+	if !reflect.DeepEqual(emptyCliNodeConfig, NodeConfigDefault) {
+		changelog, err := diff.Diff(emptyCliNodeConfig, NodeConfigDefault)
```

Comment:
yeah actually initially i was using cmp.Diff only but i am running into this issue with it

`cannot handle unexported field at {main.NodeConfig}.Node.BatchPoster.gasRefunder:
	"github.com/offchainlabs/nitro/arbnode".BatchPosterConfig
consider using a custom Comparer; if you control the implementation of type, you can also consider using an Exporter, AllowUnexported, or cmpopts.IgnoreUnexported [recovered]
	panic: cannot handle unexported field at {main.NodeConfig}.Node.BatchPoster.gasRefunder:
	"github.com/offchainlabs/nitro/arbnode".BatchPosterConfig
consider using a custom Comparer; if you control the implementation of type, you can also consider using an Exporter, AllowUnexported, or cmpopts.IgnoreUnexported
`

Wasn't sure how solve it in a generic manner so that it does not happen when add a new struct, hence moved to reflect.DeepEqual, let me know you have ideas on how to solve it. Thanks.